Docker/Settings/ssl/generate-certificate.sh

```diff
@@ -8,5 +8,5 @@
 set -e
 set -x
 
-openssl dhparam -out dhparam.pem 4096
-openssl req -subj '//CN=localhost' -x509 -newkey rsa:4096 -nodes -keyout server.key -out server.crt -days 365 -addext "subjectAltName = DNS:nginx"
\ No newline at end of file
+openssl dhparam -out dhparam.pem 2048
+openssl req -subj '/CN=localhost' -x509 -newkey rsa:2048 -nodes -keyout server.key -out server.crt -days 365 -addext "subjectAltName = DNS:nginx"
```

Comment:
Double slash is incorrect, not sure why someone added that.

Min key length is 2048. 4096 is "better" protection, but (a) it takes a very long time to generate and (b) this is a self-signed certificate for demonstration purposes. We only need the certificate to be _adequate_.